sandy/sections/mf6.py

```diff
@@ -124,7 +124,7 @@ def read_mf6(tape, mat, mt):
         LAW = T.L2  # Distintion between different distribution function
         ZAP = T.C1  # Product identifier
         LIP = T.L1  # Product isomeric state identifier
-        ZAM = zam.za2zam(ZAP, meta=LIP)
+        ZAM = zam.za2zam(ZAP, meta=LIP, method=False)
```

Comment:
@EnricaBelfiore , In theory this should also work with `method="nndc"`. Why did you change it?

Reply:
I checked in the Endf6 manual and the ZAP is defined as 1000 ∗ Z + A. This works also with `method="nndc"` if meta=0, but not with higher isomeric states of the product nucleus

Reply:
In this case ZAP is always the ZA of the ground state. The metastate level is defined by LIP.
So, if I remove method does it still work?

Reply:
No because the default option is `method="nndc"`. If "nndc" is selected, even if you pass `meta=LIP`, the method `za2zam()` will return the metastate evaluated from the ZA number.  To be more clear: in line 556 of the script sandy.zam.py, we initialize `m=0` and we don't use the `meta` parameter... It could be initialized equal to the parameter `meta` but I lose the meaning of the "nncd" method for me. I left this method as the default option, as it is currently impemented in the `develop` branch